doc/api/deprecations.md

```diff
@@ -807,6 +807,15 @@ Importing assert directly is not recommended as the exposed functions will use
 loose equality checks. Use `require('assert').strict` instead. The API is the
 same as the legacy assert but it will always use strict equality checks.
 
+<a id="DEPXXXX"></a>
+### MakeCallback with domain property
+
+Type: Runtime
+
+Users of `MakeCallback` that add the `domain` property to carry context,
+should start using the `async_context` variant of `MakeCallback` or the
+`AsyncResource` class.
```

Comment:
… or `CallbackScope`?

Reply:
minor nit, `DEP00XX` in order for the tooling to catch these on release builds (there's a script that looks for the `DEP00XX` pattern specifically)